Import explodes on Python 3.7

```python
import bblfsh.github.com.gogo.protobuf.gogoproto.gogo_pb2
```

fails with

```python
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python3.7/site-packages/bblfsh/github/com/gogo/protobuf/gogoproto/gogo_pb2.py", line 670, in <module>
    google_dot_protobuf_dot_descriptor__pb2.EnumOptions.RegisterExtension(goproto_enum_prefix)
  File "/usr/local/lib/python3.7/site-packages/google/protobuf/internal/python_message.py", line 751, in RegisterExtension
    cls.DESCRIPTOR.file.pool.AddExtensionDescriptor(extension_handle)
  File "/usr/local/lib/python3.7/site-packages/google/protobuf/descriptor_pool.py", line 264, in AddExtensionDescriptor
    extension.containing_type.full_name, extension.number))
AssertionError: Extensions "gogoproto.goproto_enum_prefix" and "gogoproto.goproto_enum_prefix" both try to extend message type "google.protobuf.EnumOptions" with field number 62001.
```

Reproducible exclusively on Python 3.7.0 final (works fine with beta3). How to reproduce:

```
docker run -it --rm python:3.7.0-stretch bash
pip3 install bblfsh
python3 -c "import bblfsh.github.com.gogo.protobuf.gogoproto.gogo_pb2"
```

Related to https://github.com/google/protobuf/issues/2533 but not really.

Looks like the top-level import of `bblfsh` is not visible to the import system and it imports the module twice. The workaround is to comment out that assertion inside protobuf.

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,5 +1,6 @@
 language: python
 sudo: true
+dist: xenial
 services:
   - docker
 cache:
@@ -8,6 +9,7 @@ cache:
 python:
   - "3.5"
   - "3.6"
+  - "3.7"
 install:
   - wget https://github.com/bblfsh/client-python/releases/download/v2.2.1/protobuf-python_3.4.1-1_amd64.deb
   - sudo dpkg -i protobuf-python_3.4.1-1_amd64.deb

--- a/setup.py
+++ b/setup.py
@@ -1,3 +1,4 @@
+import fileinput
 import logging
 import os
 import pkg_resources
@@ -81,7 +82,8 @@ def cpr(src, dst):
     src = src.format(**FORMAT_ARGS)
     dst = dst.format(**FORMAT_ARGS)
     log.info("cp -pr %s %s", src, dst)
-    shutil.rmtree(dst, ignore_errors=True)
+    if os.path.isdir(dst):
+        shutil.rmtree(dst)
     shutil.copytree(src, dst, symlinks=True)
 
 
@@ -161,8 +163,9 @@ def proto_compile():
         log.info("%s -m grpc.tools.protoc " + " ".join(main_args), sys.executable)
         protoc_module.main(main_args)
 
+    sdk_root = j("bblfsh", "gopkg", "in", "bblfsh", "sdk", SDK_MAJOR)
     # SDK
-    protoc(j("bblfsh", "gopkg", "in", "bblfsh", "sdk", SDK_MAJOR, "protocol"),
+    protoc(j(sdk_root, "protocol"),
            j("gopkg.in", "bblfsh", "sdk." + SDK_MAJOR, "protocol", "generated.proto"),
            "-I" + j("gopkg.in", "bblfsh", "sdk." + SDK_MAJOR, "protocol"))
     # UAST
@@ -170,6 +173,12 @@ def proto_compile():
            grpc=False)
     protoc("bblfsh", j("gopkg.in", "bblfsh", "sdk." + SDK_MAJOR, "uast", "generated.proto"),
            grpc=False)
+    for line in fileinput.input([j(sdk_root, "protocol", "generated_pb2.py"),
+                                 j(sdk_root, "uast", "generated_pb2.py")],
+                                inplace=True):
+        print(line.replace("from github.com.gogo.protobuf.gogoproto import",
+                           "from bblfsh.github.com.gogo.protobuf.gogoproto import"),
+              end="")
 
 
 def do_get_deps():

